docs/en/getting-started/get-started-docker.asciidoc

```diff
@@ -179,7 +176,7 @@ endif::[]
 --
 ["source","sh"]
 ----
-docker-compose down
+docker-compose stop
```

Comment:
`down` will tear down the containers so they have to be rebuilt, since we only need to restart them it would be better to use `stop`